.github/workflows/close-stale-issues.yml

```diff
@@ -0,0 +1,24 @@
+#
+# Docs: https://github.com/marketplace/actions/close-stale-issues
+#
+name: Close Stale Issues
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-issue-message: 'This issue is being marked stale because it has been open for 80 days with no activity. Please comment if this issue is still affecting you. If there is no change, this issue will be closed in 20 days.'
+          stale-pr-message: 'This PR is being marked stale because it has been open for 80 days with no activity. Please upadte the PR or ask for a fresh review'
+          close-issue-message: 'This issue was closed because it became stale. If the issue is still affecting you, please re-open it, or file a fresh Issue with updated information.'
+
+          days-before-stale: 80
+          days-before-close: 100
```

Comment:
Looking at the docs:

```

days-before-close | Idle number of days before closing an stale issue/PR.Defaults to 7.
-- | --

```

So we should have 20 in this case? otherwise it will take a total of 180 days to close an issue?

Reply:
Looking at the example:

```
name: 'Close stale issues and PRs'
on:
  schedule:
    - cron: '30 1 * * *'

jobs:
  stale:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/stale@v3
        with:
          stale-issue-message: 'This issue is stale because it has been open 30 days with no activity. Remove stale label or comment or this will be closed in 5 days.'
          days-before-stale: 30
          days-before-close: 5
```

Look like we need to change 100 to 20

Reply:
good point - thank you! 

I've tweaked the numbers to make the stale-tagging sooner, and give longer for updates